spec/pb_kits/playbook/props/base_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Playbook
+  module Props
+    describe Base do
+      describe "#validate!" do
+        it "returns true when child class passes validation" do
+          class TruthyValidationClass < Playbook::Props::Base
+            def validate(_value); true; end
+          end
+
+          expect(TruthyValidationClass.new.validate!(nil)).to eq true
+        end
+
+        it "returns true when child class does not contain custom validation" do
+          class NoValidationClass < Playbook::Props::Base; end
```

Comment:
Same as above. I would also consider dropping the inheritance all together, and be more direct that the value return true.